src/lspmgt_frontend/.gitignore

```diff
@@ -20,3 +20,5 @@ npm-debug.log*
 yarn-debug.log*
 
 yarn-error.log*
+
+/semantic
```

Comment:
Why did you add the whole `/semantic` directory to the .gitignore?

Reply:
I was playing around with .gitignore with that folder specifically. I removed it and should not be there on my next pull request, good catch.

Reply:
Gotcha 👍